youtube_dl/extractor/intldropout.py

```diff
@@ -0,0 +1,152 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .vimeo import VHXEmbedIE
+
+from ..utils import (
+    ExtractorError,
+    sanitized_Request,
+    urlencode_postdata,
+)
+
+import re
+
+# https://intl.dropout.tv/login
+# GET
+# authenticity_token
+
+# https://intl.dropout.tv/login
+# POST
+# authenticity_token
+# email
+# password
+# utf8 ✓
+
+
+# https://embed.vhx.tv/videos/414462?api=1&autoplay=1&referrer=https%3A%2F%2Fintl.dropout.tv%2Fbrowse&playsinline=1&title=0&context=https%3A%2F%2Fintl.dropout.tv%2Fbrowse&back=Browse&color=feea3b&sharing=1&auth-user-token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo1Mzk0NDEwLCJleHAiOjE1NDc0NzA1NDB9._y4H94pKyIOu_GT11qC2SeJnSou6EzN9jI1A-P3tbo8&live=0
+# https://vhx-adaptive-hap.akamaized.net/-ctx--user_id,5394410--platform_id,27--video_id,414462--channel_id,55407--plan,standard-/vods3cf/0/amlst:c-55407/v-414462/2220471,2220472,2220473,2220474,2220475,2220476/playlist.m3u8?token=exp=1547481565~acl=/-ctx--user_id,5394410--platform_id,27--video_id,414462--channel_id,55407--plan,standard-/vods3cf/0/amlst:c-55407/v-414462/2220471,2220472,2220473,2220474,2220475,2220476/*~hmac=ceb8508146d2dec2b868db9ca304ec13d54502cca0a7d1cd0def7a85a9ef3962&
+# https://api.vhx.tv/videos/414462/files?auth_user_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo1Mzk0NDEwLCJleHAiOjE1NDc0NzA1NDB9._y4H94pKyIOu_GT11qC2SeJnSou6EzN9jI1A-P3tbo8&_=1547463565300
+
+
+class IntlDropoutIE(VHXEmbedIE):
+    IE_NAME = 'intldropout'
+    IE_DESC = 'International Dropout.tv'
+    _NETRC_MACHINE = 'intl.dropout.tv'
```

Comment:
Must be dropouttv.

Reply:
i would suggest `intldropouttv`. just to make sure it doesn't conflict with `dropout.tv` which is a different Site for US Users